javascript/ql/src/semmle/javascript/ApiGraphs.qll

```diff
@@ -849,31 +849,39 @@ private module Label {
   string promised() { result = "promised" }
 }
 
+private class NodeModuleSourcesNodes extends DataFlow::SourceNode::Range {
+  NodeModuleSourcesNodes() {
+    exists(NodeModule m |
+      this = DataFlow::ssaDefinitionNode(SSA::implicitInit([m.getModuleVariable(), m.getExportsVariable()]))
```

Comment:
Would it be worth binding the variable to a field and exposing it through a getter? Then we could use that getter in the charpred of `{Module,Exports}VarNode` to avoid some duplication.